tamr_client/instance/backup.py

```diff
@@ -0,0 +1,94 @@
+from tamr_client import response
+from tamr_client._types import Instance, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class AlreadyCanceled(TamrClientException):
+    """Raised when attempting to cancel a backup that has already failed or been canceled.
+    """
+
+    pass
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing a backup that does not exist on the server.
+    """
+
+    pass
+
+
+def get(session: Session, instance: Instance) -> list:
```

Comment:
We should model backups as a type.

Also: `list` is the list constructor in Python, but not a valid type for type-checking. Use `typing.List` instead.

```suggestion
def get(session: Session, instance: Instance) -> List[Backup]:
```